change-notes/1.18/analysis-cpp.md

```diff
@@ -33,4 +33,4 @@
  
 ## Changes to QL libraries
 
-* *Series of bullet points*
+* Added a hash consing library for structural comparison of expressions.
```

Comment:
Please refer readers to where they can find the library. Maybe instead write "Added a new library `semmle.code.cpp.valuenumbering.HashCons` for structural comparison of expressions."

Also, this file is in conflict.